day-01/day-01-main.cc

```diff
@@ -1,15 +1,59 @@
 // Advent of Code 2020
 
-#include <iostream>
+#include <unordered_map>
 
-#include "../helper-classes/list-reader.h"
+#include "../helper-classes/int-list-reader.h"
```

Comment:
Generally I like to see includes relative to the source root, e.g. in this case just remove the `../`